budgetportal/views.py

```diff
@@ -173,11 +173,11 @@ def department_list_csv(request, financial_year_id, spheres=["national", "provin
             for department in government.departments.all():
                 writer.writerow(
                     {
-                        "government": government.name.encode("utf-8"),
-                        "department_name": department.name.encode("utf-8"),
+                        "government": str(government.name.encode("utf-8")),
```

Comment:
Would it make sense to remove the `.encode` (and then the `str()`) here, since it's already str coming out of the DB?

Reply:
yeah, thats what I'm just doing after a bit of digging into the code. It seems that normal strings are enough for this project.

Thank you! @jbothma